docsrc/markdown/application_service.md

```diff
@@ -0,0 +1,34 @@
+# Application Service
```

Comment:
Please reference this document in https://github.com/SymphonyPlatformSolutions/symphony-api-client-python/blob/2.0/docsrc/index.md so that it is visible in the main documentation page